schema/src/BT01/BT01CreditClass.yaml

```diff
@@ -0,0 +1,155 @@
+id: https://framework.regen.network/schema/BTCreditClass
+name: BTCreditClass
+prefixes:
+  linkml: https://w3id.org/linkml/
+  rfs: https://framework.regen.network/schema/
+  schema: http://schema.org/
+imports:
+  - linkml:types
+default_curi_maps:
+  - semweb_context
+default_prefix: rfs
+default_range: string
+
+classes:
+  BT01CreditClass:
+    class_uri: rfs:BT01CreditClass
+    slots:
+      - approvedMethodologies
+      - coBenefits
+      - creditGenerationMethod
+      - creditingTerm
+      - description
+      - ecosystemType
+      - eligibleActivities
+      - geographicApplicability
+      - name
+      - permanencePeriod
+      - primaryImpact
+      - sourceRegistry
+      - url
+  Methodology:
+    class_uri: rfs:Methodology
+    description: Details about a specific methodology.
+    attributes:
+      identifier:
+        description: Unique identifier for the methodology.
+      name:
+        description: Name of the methodology.
+      url:
+        description: URL to the methodology document.
+      version:
+        description: Version number of the methodology.
+  MethodologyList:
+    class_uri: rfs:MethodologyList
+    description: List of methodologies for credit generation.
+    attributes:
+      itemListElement:
+        range: Methodology
+        multivalued: true
+        description: "List of methodology items."
+  PrimaryImpact:
+    class_uri: rfs:PrimaryImpact
+    description: Primary impact of the biodiversity project.
+    attributes:
+      id:
+        slot_uri: rfs:id
+      name:
+        slot_uri: schema:name
+      SDGs:
+        description: List of relevant Sustainable Development Goals for this impact.
+        range: SDG
+        multivalued: true
```

Comment:
@S4mmyb currently `SDGs` as a slot name doesn't fit our linkml linting configuration (slots should be lower camel case). What about just `SDG`?

In the past I have tried to avoid using plural wording for technical properties and just stick with singular words everywhere, knowing that some singular properties might indeed have mutliple values, thinking that the schema will specify if they are multi-valued, not the naming itself. Curious what you think about this more generally.